src/ViewHelpers/OrViewHelper.php

```diff
@@ -65,7 +65,7 @@ public static function renderStatic(array $arguments, \Closure $renderChildrenCl
 
         $content = $renderChildrenClosure();
 
-        if (null === $content) {
+        if ('' === $content) {
```

Comment:
Now if `$content` really is `null`, the viewhelper would not work as expected anymore.

Thus one would actually also check against `null` here. Notice that `empty()` should not be used since the viewhelper would also render the alternative on `'0'` as `$content` then.

In any case I expect this to be breaking due to a change of behavior (even if desired). So this is only an option for the next major version.